### PR TITLE
Adds shipper details to parent row

### PR DESCRIPTION
**Pull request by ACC25**

### CODE REVIEW

File: src/gateway-types/types.ts [hunk 1]
```diff
@@ -1,6 +1,9 @@
 import {UrlAttachment} from "../formatter/types";
 import {Methodology, WhiteGloveServiceType} from "./constants";
 
+// Keep type definitions as concise as possible to reduce work as Gateway types evolve
```
Comment (trialbyerror) [inline src/gateway-types/types.ts]: 👍

File: README.md [hunk 2]
```diff
@@ -1,9 +1,4 @@
-# AWS Lambda with Typescript starter kit 
-
-Fork this repo to write lambda functions on AWS using Typescript. This repo only deals with writing the function itself
-and not the deployment process. For deployment you can use tools such as Jenkins or
-[node-lambda](https://www.npmjs.com/package/node-lambda).
-
+# Gateway APIs Push to Smartsheet Lambda
```
Comment (trialbyerror) [inline README.md]: 👍

File: README.md [hunk 3]
```diff
@@ -12,21 +7,8 @@ and not the deployment process. For deployment you can use tools such as Jenkins
...
-This package is used to enforce consistent coding style. There is also a pre-commit hooks enabled to run TS-lint 
-before every commit.
-
-### Winston
-There is a logger module developed on top of Winston package for logging purposes. 
-
-
++ `.env` file is necessary for function to run properly
++ Refer to the AWS Lambda Management Console for Env variables
++ Running locally with pull from AWS SQS and push to Smartsheet
```
Comment (trialbyerror) [inline README.md]: Can we add a comment that the default local configuration is to use the dev SQS and dev Smartsheet, and therefore the exact keys and values for dev should be copied from the Management Console?

Comment (ACC25) [reply README.md]: 👍 Added comment

**Pull request merged**